docs/next/en-US/state/revalidation.md

```diff
@@ -10,15 +10,15 @@ The first type of revalidation is the simplest: you set a schedule with `.revali
 
 ## Logic-based revalidation
 
-When you have more stringent needs, you might wish to use logic-based revalidation, which is based on the `.should_revalidate_fn()` method on `Template`. To this, you provide an `async` function of the usual sort with the usual `BlamedError<E>` error handling (see [here](:state/build) for an explanation of that) that takes a [`StateGeneratorInfo`](=prelude/struct.StateGeneratorInfo@perseus) instance and the user's request, and you return a `bool`: if it's true, the page will revalidate, but, if `false`, the old state will stand. This can be used to do more advanced things like having a database of new news, but also having a micro-site set to tell you whether or not there is new news. Thus, you can perform the quicker check to the micro-site (which acts as a [canary](https://en.wikipedia.org/wiki/Sentinel_species)) to avoid unnecessary revalidations, which will improve performance.
+When you have more stringent needs, you might wish to use logic-based revalidation, which is based on the `.should_revalidate_fn()` method on `Template`. To this, you provide an `async` function of the usual sort with the usual `BlamedError<E>` error handling (see [here](:state/build) for an explanation of that) that takes a [`StateGeneratorInfo`](=prelude/struct.StateGeneratorInfo@perseus) instance and the user's request, and you return a `bool`: if it's true, the page will revalidate, but, if `false`, the old state will stand. This can be used to do more advanced things like having a database of new news, but also having a micro-site set to tell you whether there is new news. Thus, you can perform the quicker check to the micro-site (which acts as a [canary](https://en.wikipedia.org/wiki/Sentinel_species)) to avoid unnecessary revalidations, which will improve performance.
 
-Using both logic-based revalidation *and* time-based revalidation is perfectly permissible, as the logic-based revalidation will only be executed on the interval of the time-based. For our news site, therefore, we might want to use the logic-based revalidation to check a canary as to whether or not there is any new news, and then only run that check hourly. This would lead to hourly checks of whether or not we *should* revalidate, rather than just blindly doing so, which can improve performance greatly.
+Using both logic-based revalidation *and* time-based revalidation is perfectly permissible, as the logic-based revalidation will only be executed on the interval of the time-based. For our news site, therefore, we might want to use the logic-based revalidation to check a canary as to whether there is any new news, and then only run that check hourly. This would lead to hourly checks of whether we *should* revalidate, rather than just blindly doing so, which can improve performance greatly.
```

Comment:
```suggestion
Using both logic-based revalidation *and* time-based revalidation is perfectly permissible, as the logic-based revalidation will only be executed on the interval of the time-based. For our news site, therefore, we might want to use the logic-based revalidation to check a canary as to whether or not there is any new news, and then only run that check hourly. This would lead to hourly checks of whether or not we *should* revalidate, rather than just blindly doing so, which can improve performance greatly.
```

A few more whether or nots.